### PR TITLE
pep8 pass over the codebase

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,5 @@ install:
   - "pip install -r requirements.txt --use-wheel"
   - "pip install -r requirements-dev.txt --use-wheel"
 script:
-  - python manage.py syncdb --noinput
-  - python manage.py migrate
-  - python manage.py test
+  - flake8 .
```
Comment (smn) [inline .travis.yml:11]: @imsickofmaps please, for the love of everything turning red in our editors, automate the pep8 checks in your $EDITOR.

**Comment by smn** [comment]
@imsickofmaps ready for review

File: snappybouncer/views.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
 from django.shortcuts import render
 
 # Create your views here.
+render
```
Comment (imsickofmaps) [inline snappybouncer/views.py]: I'll just empty this file

File: subsend/models.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
 from django.db import models
 
 # Create your models here.
+models
```
Comment (imsickofmaps) [inline subsend/models.py]: And empty this one

**Comment by imsickofmaps** [comment]
:+1:

**Comment by smn** [comment]
Sometimes Django gets unhappy when there isn't a models.py file, I recall there being a time when it wasn't considered an 'app' if it didn't have models.py file.

**Pull request merged**